site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -136,13 +136,11 @@ key={("https", "retail.example"),
 
 #### First-Party Sets and cookie partitioning
 
-[First-Party Sets](/blog/first-party-sets-sameparty/#how-to-define-a-first-party-set) allow related domain names that are owned and operated by the same entity to be treated as the same first-party as the top-level site, in situations where Chrome may apply restrictions, such as access to cookies on third-party subresources.
+[First-Party Sets](/blog/first-party-sets-testing-instructions) is a web platform mechanism for developers to declare relationships among sites, so that browsers can use this information to enable limited cross-site cookie access for specific, user-facing purposes. Chrome will use these declared relationships to decide when to allow or deny a site access to their cookies when in a third-party context.
 
-When an embedded service sets a cookie on a site that's a member of a First-Party Set, that cookie will be accessible to the service when it's embedded on any of the First-Party Set member sites. This allows the embedded service to provide a seamless, unified user session across all member sites.
+First-Party Sets design relies on Storage Access API and does not integrate with CHIPS partitioning.
 
-Sites from the same First-Party Set will have the same partition key—the owner of the set. 
-
-{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/ho7jqWrZzBxmX3fsIpTt.png", alt="", width="800", height="548" %}
+If you have a use case that relies on integration of CHIPS and FPS you can (provide examples and feedback on the GitHub issue)[https://github.com/WICG/first-party-sets/issues/94].
```

Comment:
1. Link is not formatted correctly (square brackets/parentheses interchanged).
2. The acronym "FPS" is mentioned for the first time here. Can you add it next to the full spelling earlier in this section?
3. Might be better to say "a shared cookie partition across sites within an FPS" instead of  "integration of CHIPS and FPS".